scripts/src/javatests/org/oppia/android/scripts/maven/MavenDependenciesListWriterTest.kt

```diff
@@ -0,0 +1,338 @@
+//    val exception = assertThrows(Exception::class) {
+//      MavenDependenciesListWriter.main(
+//        arrayOf(
+//          "${tempFolder.root}",
+//          "scripts/assets/maven_install.json",
+//          "${tempFolder.root}/scripts/assets/maven_dependencies.pb"
+//        )
+//      )
+//    }
+//
+//    assertThat(exception).hasMessageThat().contains(COMPLETE_LICENSE_DETAILS_MESSAGE)
+  }
+
+  @Test
+  fun testMockitoUnitTest() {
+    assertThat(mockNetworkAndBazelUtils.scrapeText("https://www.google.com"))
+      .contains("passed")
+  }
+
+  private fun getMavenDependency(
+    artifactName: String,
+    version: String,
+    licenseList: List<License> = listOf<License>()
+  ): MavenDependency {
+    return MavenDependency
+      .newBuilder()
+      .setArtifactName(artifactName)
+      .setArtifactVersion(version)
+      .addAllLicense(licenseList)
+      .build()
+  }
+
+  private fun getMavenDependencyList(
+    dependencyNamesList: List<DependencyName>
+  ): MavenDependencyList {
+    val dependenciesList = mutableListOf<MavenDependency>()
+    dependencyNamesList.forEach { dependencyName ->
+      when (dependencyName) {
+        DependencyName.FIREBASE_ANALYTICS ->
+          dependenciesList.add(
+            getMavenDependency(
+              artifactName = "com.google.firebase:firebase-analytics:17.5.0",
+              version = "17.5.0"
+            )
+          )
+        DependencyName.PROTO_LITE ->
+          dependenciesList.add(
+            getMavenDependency(
+              artifactName = "com.google.protobuf:protobuf-lite:3.0.0",
+              version = "3.0.0"
+            )
+          )
+        DependencyName.GLIDE_ANNOTATIONS ->
+          dependenciesList.add(
+            getMavenDependency(
+              artifactName = "com.github.bumptech.glide:annotations:4.11.0",
+              version = "4.11.0"
+            )
+          )
+        DependencyName.IO_FABRIC ->
+          dependenciesList.add(
+            getMavenDependency(
+              artifactName = "io.fabric.sdk.android:fabric:1.4.7",
+              version = "1.4.7"
+            )
+          )
+        else -> dependenciesList.add(
+          getMavenDependency(
+            artifactName = "androidx.databinding:databinding-adapters:3.4.2",
+            version = "3.4.2"
+          )
+        )
+      }
+    }
+    return MavenDependencyList
+      .newBuilder()
+      .addAllMavenDependency(dependenciesList)
+      .build()
+  }
+
+  private fun getLicense(
+    licenseName: String,
+    primaryLink: String = "",
+    primaryLinkType: PrimaryLinkType = PrimaryLinkType.UNSPECIFIED,
+    alternativeLink: String = ""
+  ): License {
+    return License
+      .newBuilder()
+      .setLicenseName(licenseName)
+      .setPrimaryLink(primaryLink)
+      .setPrimaryLinkType(primaryLinkType)
+      .setAlternativeLink(alternativeLink)
+      .build()
+  }
+
+  private fun writeMavenInstallJson(file: File) {
```

Comment:
add kdoc

Reply:
it was no more needed and hence removed.

Reply:
@prayutsu I think this hasn't been resolved yet. PTAL.

Reply:
Done.